setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='edx-opaque-keys',
-    version='0.3.2',  # Please update CHANGELOG when you bump versions
+    version='0.3.1',
```

Comment:
It actually won't until we tag the repo - that's when the automatic Travis pkg build/upload would occur.